bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -73,11 +77,12 @@ def _is_security_enabled(self, config):
             return False
 
     def _setup_cluster_and_execute_test_config(self, config):
```

Comment:
What's the convention for private methods? I think we've been doing two `__`

Reply:
Not written by me but I can change if @setiah is okay with that or not making changes on his side

Reply:
You may skip this. This (and more) will be refactored in a separate PR which adds UTs as well